test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -47,15 +72,60 @@
    * The default escape character is disabled, to match the DQL
    * behavior (and standard SQL).
    */
+  static {
+    Database dbname = Database.H2;
+    String dbdriver = "org.h2.Driver";
+    String dburl = "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DEFAULT_ESCAPE=";
+    String dbuser = "sa";
+    String dbpassword = "";
+
+    String propertiesFile = System.getProperty("build.properties");
+    if (!Strings.isNullOrEmpty(propertiesFile)) {
+      try (FileInputStream fis =
```

Comment:
How about the more concise
try (BufferedReader = Files.newBufferedReader(propertiesFile, UTF_8)) {
and drop the new InputStreamReader below.